tests/data/cli/compare/virt-install-kvm-loongarch64-basic.xml

```diff
@@ -0,0 +1,93 @@
+<domain type="kvm">
+  <name>fedora19</name>
+  <uuid>00000000-1111-2222-3333-444444444444</uuid>
+  <metadata>
+    <libosinfo:libosinfo xmlns:libosinfo="http://libosinfo.org/xmlns/libvirt/domain/1.0">
+      <libosinfo:os id="http://fedoraproject.org/fedora/19"/>
+    </libosinfo:libosinfo>
+  </metadata>
+  <memory>65536</memory>
+  <currentMemory>65536</currentMemory>
+  <vcpu>2</vcpu>
+  <os>
+    <type arch="loongarch64">hvm</type>
+    <boot dev="network"/>
+  </os>
+  <features>
+    <acpi/>
+  </features>
+  <clock offset="utc"/>
+  <devices>
+    <emulator>/usr/bin/qemu-system-loongarch64</emulator>
+    <controller type="usb" model="qemu-xhci" ports="15"/>
+    <interface type="bridge">
+      <source bridge="testsuitebr0"/>
+      <mac address="00:11:22:33:44:55"/>
+      <model type="virtio"/>
+    </interface>
+    <console type="pty"/>
+    <channel type="unix">
+      <source mode="bind"/>
+      <target type="virtio" name="org.qemu.guest_agent.0"/>
+    </channel>
+    <channel type="spicevmc">
+      <target type="virtio" name="com.redhat.spice.0"/>
+    </channel>
+    <input type="tablet" bus="usb"/>
+    <input type="keyboard" bus="usb"/>
+    <graphics type="spice" port="-1" tlsPort="-1" autoport="yes">
+      <image compression="off"/>
+    </graphics>
+    <sound model="ich6"/>
```

Comment:
`ich6` is an Intel-specific device that also just so happens to be conventional PCI, which means that this will force libvirt to add a `pcie-to-pci-bridge` controller. Not ideal.

I think `virtio-sound` might be a better default for non-x86 architectures, including existing ones. However, it doesn't look like support for it is currently implemented in libvirt, which clearly makes it a non-starter...

If `ich6` actually works on loongarch64 and friends (does it?) I guess it's as good an interim solution as any. Or would it be better to have no sound at all? Probably not.

Reply:
loongarch64 uses ich6 to have sound, but the sound may not be continuous due to insufficient performance. I see other non-x86 architectures use ich9(The sound effects of ich9 and ich6 are the same),  For the next push, I will change the sound device to ich9,We can modify this together when libvirt supports virtio-sound.